components/fieldlabel/index.css

```diff
@@ -117,7 +134,7 @@ governing permissions and limitations under the License.
     flex-direction: column;
 
     & + .spectrum-Form-item {
-      margin-block-start: var(--spectrum-tableform-labelsabove-item-gap);
+      margin-block-start: var(--spectrum-spacing-100);
```

Comment:
Any reason why this little fella isn't a component-level token above? It's not forbidden, per se, but I'd rather see it defined above for consistency.

Reply:
@lazd I am actually not sure which item to use here. 
Spacing 100 would be 
```JSON
  "spacing-100": {
    "value": "8px"
  },
  ```
  
  Maybe use this? I don't see any other component level tokens for field-label matching 8px.

```JSON
  "field-label-top-to-asterisk-small": {
    "sets": {
      "desktop": {
        "value": "8px"
      },
      "mobile": {
        "value": "11px"
      }
    }
  },
  ```

Reply:
I think you just need to define that as a component-level token in CSS:

```
.spectrum--medium {
  --spectrum-field-label-top-to-asterisk-small: 8px;
}
.spectrum--large {
  --spectrum-field-label-top-to-asterisk-small: 11px;
}
```

Reply:
Added, thanks @lazd